### ISSUE TITLE
Feature request: add 'take 2 creds from mumba temple' as an option

### ISSUE THREAD

**Issue by Zach-ER** [mtgred/netrunner#1455]
It's so fiddly atm.

**Comment by FarCryFromHuman** [comment]
Might be better to just add a "take all" option to all cards with recurring credits. This would be possible programmatically instead of having to add the option to every card manually.

**Comment by queueseven** [comment]
Or we should remove the "Derez", so that you can take a credit with a single click instead of 2. Who uses "Derez" anyway? It could then be done with <kbd>CTRL</kbd> + Click or something like that.

**Comment by Quarg** [comment]
Perhaps remove the derez on things and add a /derez command.

**Comment by FarCryFromHuman** [comment]
I like @Quarg approach best as it will help alleviate confusion with newer players. I have had players derez cards because they thought they could, but this is not allowed by the rules. The only real reason for this prompt is to revert miclick rezzes or rewind the gamestate (which could easily be resolved via the command).

I also think it would be nice to have take-all functionality on any card with more than 1 recurring credit. Multithreader decks would greatly appreciate this, and it would play nicely from a UI perspective with #1279 if anyone implements this.

**Comment by zaroth** [comment]
If we go with @queueseven's idea, we could make a `[Ctrl|Alt|Shift] + click` as a "quick action" for many cards (and denote them as such when being listed in the pop-up menu). If we keep consistency, this could be a great boon to the UX.

Example with Gordian Blade:
* LMB: brings up a menu:
```
x cred: [S] match strength of current ICE
1 cred: [C] break 1 code gate subroutine
1 cred: strength +1
```

* Shift+LMB: matches strength of current ICE
* Ctrl+LMB: breaks 1 code gate subroutine

Similarly with Mumba Temple:
* LMB: brings up a menu:
```
[C] take 1 credit
[S] take all credits
derez
```
* Ctrl+LMB: takes 1 credit
* Shift+LMB: takes all credits

Regarding merits of removing `Derez` as one of easily accessible options, it's a separate issue. IMHO it should stay, as the console commands are a crutch and everything should be possible to do from the clickable UI. To help the new player's confusion, the commands meant to be only for fixing stuff could be marked with separate style (e.g. yellow background and a wrench) and/or displayed only after a "fix gamestate mode" is enabled (e.g. via checkbox, hidden purely via JS on the client side).

**Comment by FarCryFromHuman** [comment]
As long as the gamestate fixing commands are hidden or called out in some fashion, either solution is fine. Confusing new players is certainly my primary concern.

Hotkeys would be nice while playing on anything with a keyboard (which I assume is the bulk of the playerbase; I can't imagine tablets have caught on that strongly).

I do have to say, as a player, that I appreciate when a command is hidden/greyed-out/otherwise disabled when it is not available per the rules. This was one of my biggest gripes about the OCTGN implementation and a large part of why I switched to J.net. I'd like to preserve the simplicity of the interface, but that can't come at the cost of poor rules enforcement. Luckily, at least in this case, I think we can have both.

**Comment by Saintis** [comment]
Adding a "take all credits" option should be the easiest fix to implement.

**Comment by NoahTheDuke** [comment]
Implemented with #4262
